custom_components/tahoma/cover.py

```diff
@@ -157,7 +157,10 @@ def current_cover_position(self):
         if position is None or position < 0 or position > 100:
             return None
 
-        if "Horizontal" not in self.device.widget:
+        if (
+            "Horizontal" not in self.device.widget
+            and "AwningValance" != self.device.widget
```

Comment:
This should be added to all mentions of "Horizontal" in `cover.py`, if I am not mistaken.

Reply:
done